torchao/quantization/quantize_/int4/int4_groupwise_preshuffle_tensor.py

```diff
@@ -36,10 +40,16 @@ class Int4GroupwisePreshuffleTensor(TorchAOBaseTensor):
 
     Tensor Attributes:
         packed_weight: packed int4 weight, either 2D (N, K/2) or 3D (B, N, K/2), last dimension is packed
-        group_scale: (K/group_size, N) for 2D Tensor, (B, N, K/group_size) for 3D Tensor
-               dtype is the same as the original Tensor dtype
-        group_zero: (K/group_size, N) for 2D Tensor, (B, N, K/group_size) for 3D Tensor
-               dtype is the same as the original Tensor dtype
+        for bf16 activation:
```

Comment:
are there expectations on strides for these two different versions?

Reply:
I think all of these should be contiguous